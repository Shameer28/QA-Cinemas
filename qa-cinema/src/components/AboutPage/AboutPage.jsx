import './AboutPage.css'
import Image from 'react-bootstrap/Image'
import Row from 'react-bootstrap/Row'
import Col from 'react-bootstrap/Col'
import image1 from './../img/aboutus.png'
import image2 from './../img/oliver.jpg'
import image3 from './../img/haaris.jpg'
import image7 from './../img/andrew.jpg'
import image4 from './../img/shameer.jpg'
import image5 from './../img/social.jpg'
import Container from 'react-bootstrap/Container';

const AboutPage = () => {
    return (
        <div class="background">

            <Image src={image1} alt="About us photo" width="100%" fluid /><br />
            <div style={{ margin: "auto", width: "50%" }}>
                <Container>
                    <div>
                        <h1>Meet the Team</h1><br />
                        <Container>
                            <Row>
                                <Col xs={6} md={4}>
                                    <Image src={image4} width="100%" alt="Shameer's Photo" thumbnail />
                                    <p><b>Name</b> : Shameer Dar<br /></p>
                                    <b>Role</b> : Front-End Developer<br />
                                </Col>
                                <Col xs={6} md={4}>
                                    <Image src={image5} width="100%" alt="LinkedIn & GitHub logos" thumbnail />
                                    <p><b>Social Media:</b><br />
                                        LinkedIn: <a href="https://www.linkedin.com/in/shameer-dar-91b120a5/" target="_blank">https://www.linkedin.com/in/shameer</a><br />
                                        GitHub: <a href="https://github.com/Shameer28" target="_blank">https://github.com/Shameer28</a></p>
                                </Col>
                            </Row>
                        </Container>
                        <br />
                        <Container>
                            <Row>
                                <Col xs={6} md={4}>
                                    <Image src={image3} width="100%" height="100%" alt="Haaris's Photo" thumbnail />
                                    <p><b>Name</b> : Haaris Moghal <br /></p>
                                    <p><b>Role</b> : Front-End Developer<br /></p>
                                </Col>
                                <Col xs={6} md={4}>
                                    <Image src={image5} width="100%" alt="LinkedIn & GitHub logos" thumbnail />
                                    <p><b>Social Media:</b><br />
                                        LinkedIn: <a href="https://www.linkedin.com/in/haaris-moghal-60b8b0159/" target="_blank">https://www.linkedin.com/in/haaris</a><br />
                                        GitHub: <a href="https://github.com/HaarismoghalQA" target="_blank">https://github.com/HaarismoghalQA</a></p>
                                </Col>
                            </Row>
                        </Container>
                        <br />
                        <Container>
                            <Row>
                                <Col xs={6} md={4}>
                                    <Image src={image7} width="100%" height="100%" alt="Andrew's Photo" thumbnail />
                                    <p><b>Name</b> : Andrew McCall <br /></p>
                                    <p><b>Role</b> : Back-End Developer<br /></p>
                                </Col>
                                <Col xs={6} md={4}>
                                    <Image src={image5} width="100%" alt="LinkedIn & GitHub logos" thumbnail />
                                    <p><b>Social Media:</b><br />
                                        LinkedIn: <a href="https://www.linkedin.com/in/shameer-dar-91b120a5/" target="_blank">https://www.linkedin.com/in/shameer</a><br />
                                        GitHub: <a href="https://github.com/Andrew-McCall" target="_blank">https://github.com/Andrew-McCall</a></p>
                                </Col>
                            </Row>
                        </Container>
                        <br />
                        <Container>
                            <Row>
                                <Col xs={6} md={4}>
                                    <Image src={image2} width="100%" height="100%" alt="Oliver's Photo" thumbnail />
                                    <p><b>Name</b> : Oliver Dillion <br /></p>
                                    <p><b>Role</b> : Back-End Developer<br /></p>
                                </Col>
                                <Col xs={6} md={4}>
                                    <Image src={image5} width="100%" alt="LinkedIn & GitHub logos" thumbnail />
                                    <p><b>Social Media:</b><br />
                                        LinkedIn: <a href="https://www.linkedin.com/in/oliverdillon" target="_blank">https://www.linkedin.com/in/oliver</a><br />
                                        GitHub: <a href="https://github.com/StarLight-Oliver" target="_blank">https://github.com/StarLight-Oliver</a></p>
                                </Col>
                            </Row>
                        </Container>
                        <br />
                    </div>

                </Container>


            </div>

        </div >
    );
}

export default AboutPage;