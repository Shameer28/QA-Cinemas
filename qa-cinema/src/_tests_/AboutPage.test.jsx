import AboutPage from "../components/AboutPage/AboutPage";
import { create } from 'react-test-renderer';


describe(`About Page testing`, () => {

    it(`Should match the snapshot`, () => {
        const instance = create(<AboutPage />).toJSON();
        expect(instance).toMatchSnapshot();
    })
})