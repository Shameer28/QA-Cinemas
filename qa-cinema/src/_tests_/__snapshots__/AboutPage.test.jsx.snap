// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page testing Should match the snapshot 1`] = `
<div
  class="background"
>
  <img
    alt="About us photo"
    fluid={true}
    src="aboutus.png"
    width="100%"
  />
  <br />
  <br />
  <div>
    <div
      className="p-3 container"
    >
      <div
        className="container"
      >
        <h2
          style={
            Object {
              "color": "white",
              "fontWeight": "400",
              "fontfamily": "isonormregular, sans-serif",
              "letterSpacing": ".15em",
              "lineHeight": "1.1",
              "textTransform": "uppercase",
            }
          }
        >
          Meet our developers
        </h2>
        <br />
        <div
          style={
            Object {
              "backgroundColor": "#212529",
              "border": "solid",
              "borderColor": "black",
              "color": "white",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "80%",
            }
          }
        >
          <div
            className="row"
          >
            <div
              border="solid"
              borderColor="black"
              className="col"
            >
              <img
                alt="Shameer's Photo"
                border="solid"
                borderColor="black"
                className="img-fluid"
                src="shameer.jpg"
                width="100%"
              />
            </div>
            <div
              className="col"
              style={
                Object {
                  "margin": "auto",
                  "width": "50%",
                }
              }
            >
              <h4>
                <b>
                  Name
                </b>
                 : Shameer Dar
              </h4>
              <h4>
                <b>
                  Role
                </b>
                 : Product Owner/Front-End Developer
              </h4>
              <br />
              <div
                className="row"
              >
                <div
                  class="col-sm"
                  className="col"
                >
                  <a
                    href="https://www.linkedin.com/in/shameer-dar-91b120a5/"
                    target="_blank"
                  >
                    <img
                      alt="LinkedIn"
                      className="img-thumbnail"
                      height="100%"
                      src="linkedin.jpg"
                      width="73%"
                    />
                  </a>
                </div>
                <div
                  class="col-sm"
                  className="col"
                >
                  <a
                    href="https://github.com/Shameer28"
                    target="_blank"
                  >
                    <img
                      alt="Github"
                      className="img-thumbnail"
                      height="100%"
                      src="Github.png"
                      width="95%"
                    />
                  </a>
                  <br />
                  <br />
                </div>
                <h5
                  className="text-center"
                >
                   Contact me via LinkedIn 
                  <br />
                  or 
                  <br />
                  via Contact Us Page 
                  <br />
                  <br />
                  <a
                    className="btn btn-primary"
                    href="/contactpage"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                  >
                    Contact Us
                  </a>
                </h5>
              </div>
            </div>
          </div>
        </div>
        <br />
        <div
          style={
            Object {
              "backgroundColor": "#212529",
              "border": "solid",
              "borderColor": "black",
              "color": "white",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "80%",
            }
          }
        >
          <div
            className="row"
          >
            <div
              className="col"
            >
              <img
                alt="Haaris's Photo"
                border="solid"
                borderColor="black"
                className="img-fluid"
                height="100%"
                src="haaris.jpg"
                width="100%"
              />
            </div>
            <div
              className="col"
              style={
                Object {
                  "margin": "auto",
                  "width": "50%",
                }
              }
            >
              <h4>
                <b>
                  Name
                </b>
                 : Haaris Moghal 
                <br />
              </h4>
              <h4>
                <b>
                  Role
                </b>
                 : Front-End Developer
              </h4>
              <br />
              <div
                className="row"
              >
                <div
                  className="col"
                >
                  <a
                    href="https://www.linkedin.com/in/haaris-moghal-60b8b0159/"
                    target="_blank"
                  >
                    <img
                      alt="LinkedIn"
                      className="img-thumbnail"
                      height="100%"
                      src="linkedin.jpg"
                      width="73%"
                    />
                  </a>
                </div>
                <div
                  className="col"
                >
                  <a
                    href="https://github.com/HaarismoghalQA"
                    target="_blank"
                  >
                    <img
                      alt="Github"
                      className="img-thumbnail"
                      height="100%"
                      src="Github.png"
                      width="95%"
                    />
                  </a>
                  <br />
                  <br />
                </div>
                <h5
                  className="text-center"
                >
                   Contact me via LinkedIn 
                  <br />
                  or 
                  <br />
                  via Contact Us Page 
                  <br />
                  <br />
                  <a
                    className="btn btn-primary"
                    href="/contactpage"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                  >
                    Contact Us
                  </a>
                </h5>
              </div>
            </div>
          </div>
        </div>
        <br />
        <div
          style={
            Object {
              "backgroundColor": "#212529",
              "border": "solid",
              "borderColor": "black",
              "color": "white",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "80%",
            }
          }
        >
          <div
            className="row"
          >
            <div
              className="col"
            >
              <img
                alt="Andrew's Photo"
                border="solid"
                borderColor="black"
                className="img-fluid"
                height="100%"
                src="andrew.jpg"
                width="100%"
              />
            </div>
            <div
              className="col"
              style={
                Object {
                  "margin": "auto",
                  "width": "50%",
                }
              }
            >
              <h4>
                <b>
                  Name
                </b>
                 : Andrew McCall 
                <br />
              </h4>
              <h4>
                <b>
                  Role
                </b>
                 : Back-End Developer
              </h4>
              <br />
              <div
                className="row"
              >
                <div
                  className="col"
                >
                  <a
                    href="https://www.linkedin.com/in/shameer-dar-91b120a5/"
                    target="_blank"
                  >
                    <img
                      alt="LinkedIn"
                      className="img-thumbnail"
                      height="100%"
                      src="linkedin.jpg"
                      width="73%"
                    />
                  </a>
                </div>
                <div
                  className="col"
                >
                  <a
                    href="https://github.com/Andrew-McCall"
                    target="_blank"
                  >
                    <img
                      alt="Github"
                      className="img-thumbnail"
                      height="100%"
                      src="Github.png"
                      width="95%"
                    />
                  </a>
                  <br />
                  <br />
                </div>
                <h5
                  className="text-center"
                >
                   Contact me via LinkedIn 
                  <br />
                  or 
                  <br />
                  via Contact Us Page 
                  <br />
                  <br />
                  <a
                    className="btn btn-primary"
                    href="/contactpage"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                  >
                    Contact Us
                  </a>
                </h5>
              </div>
            </div>
          </div>
        </div>
        <br />
        <div
          style={
            Object {
              "backgroundColor": "#212529",
              "border": "solid",
              "borderColor": "black",
              "color": "white",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "80%",
            }
          }
        >
          <div
            className="row"
          >
            <div
              className="col"
            >
              <img
                alt="Oliver's Photo"
                border="solid"
                borderColor="black"
                className="img-fluid"
                height="100%"
                src="oliver.jpg"
                width="100%"
              />
            </div>
            <div
              className="col"
              style={
                Object {
                  "margin": "auto",
                  "width": "50%",
                }
              }
            >
              <h4>
                <b>
                  Name
                </b>
                 : Oliver Dillon 
                <br />
              </h4>
              <h4>
                <b>
                  Role
                </b>
                 : Scrum Master/Back-End Developer
              </h4>
              <br />
              <div
                className="row"
              >
                <div
                  className="col"
                >
                  <a
                    href="https://www.linkedin.com/in/oliverdillon"
                    target="_blank"
                  >
                    <img
                      alt="LinkedIn"
                      className="img-thumbnail"
                      height="100%"
                      src="linkedin.jpg"
                      width="73%"
                    />
                  </a>
                </div>
                <div
                  className="col"
                >
                  <a
                    href="https://github.com/StarLight-Oliver"
                    target="_blank"
                  >
                    <img
                      alt="Github"
                      className="img-thumbnail"
                      height="100%"
                      src="Github.png"
                      width="95%"
                    />
                  </a>
                  <br />
                  <br />
                </div>
                <h5
                  className="text-center"
                >
                   Contact me via LinkedIn 
                  <br />
                  or 
                  <br />
                  via Contact Us Page 
                  <br />
                  <br />
                  <a
                    className="btn btn-primary"
                    href="/contactpage"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                  >
                    Contact Us
                  </a>
                </h5>
              </div>
            </div>
          </div>
        </div>
        <br />
      </div>
    </div>
  </div>
</div>
`;
